### PR TITLE
docs: fixing links and removing empty table

### PR DESCRIPTION
**Pull request by clatapie**
As title says.

### CODE REVIEW

**Comment by clatapie** [comment]
@ansys-reviewer-bot review

**Review by RobPasMue — APPROVED** [review]
LGTM!

File: branch-name-style/action.yml [hunk 1]
```diff
@@ -31,15 +31,15 @@ description: |
...
-      - testing or test
-      - release
-      - dependabot
+      - ``fix``
+      - ``feat``
+      - ``junk``
+      - ``maint``
+      - ``docs``
+      - ``no-ci``
+      - ``testing or test``
```
Comment (RobPasMue) [inline branch-name-style/action.yml:40]:
```suggestion
      - ``testing`` or ``test``
```

**Review by SMoraisAnsys — APPROVED** [review]
LGTM I like the proposed changes on `commit-style`

**Pull request merged**